src/app/modules/consultations/consultation-profile/read-respond/read-respond.component.ts

```diff
@@ -464,6 +506,9 @@ export class ReadRespondComponent implements OnInit, AfterViewChecked {
       this.responseSubmitLoading = false;
       this.earnedPoints = response.points;
       this.consultationService.enableSubmitResponse.next(false);
+      if (this.longTextResponse) {
```

Comment:
@Nishanthh The thank you modal should be shown even if longTextResponse is false. Just check if the Consultation has questions